Maple2.Server.Game/PacketHandlers/ChangeAttributesHandler.cs

```diff
@@ -61,7 +61,7 @@ private void HandleChange(GameSession session, IByteReader packet) {
 
         lock (session.Item) {
             // Validate item being changed + lock attributes/lock item if necessary.
-            Item? item = session.Item.Inventory.Get(itemUid, InventoryType.Gear)
+            Item? item = session.Item.GetGear(itemUid)
```

Comment:
Can't you also change attribute from an gear item in your inventory?

Reply:
getgear check inventory and equipped